background.js

```diff
@@ -16,6 +16,43 @@ function tabCallback(tabId, changeInfo, updatedTab) {
   }
 }
 
+function handleMaybeInt(maybeString) {
+  const maybeInt = parseInt(maybeString, 10);
+  if (Number.isNaN(maybeInt)) {
+    return null;
+  }
+  return maybeInt;
+}
+
+function handlePassphraseCleaning() {
+  chrome.storage.local.get(["lastModified", "lockAfterSeconds", "temporaryPassphrase"], function(data) {
+    const currentTime = Date.now();
+    const lastModified = handleMaybeInt(data.lastModified);
+    const lockAfterSeconds = handleMaybeInt(data.lockAfterSeconds);
```

Comment:
This might be a dumb question, but why would these fields be strings? It seems like they should all be numbers.